caches/partyline/src/main/java/org/commonjava/maven/galley/cache/partyline/PartyLineCacheProvider.java

```diff
@@ -68,15 +68,15 @@
 
     private List<Transfer> toDelete = Collections.synchronizedList( new ArrayList<>() );
 
-    public PartyLineCacheProvider( final File cacheBasedir, final PathGenerator pathGenerator,
+    public PartyLineCacheProvider( final File cacheBasedir, final Boolean timeoutProcessing, final PathGenerator pathGenerator,
```

Comment:
Recommend to give another constructor with this param, and leave this one with a default value of this timeoutProcessing to call the new one. This can make the constructor compatible with other lib without any changes.(if possible, or if this change is really needed)